plugins/quickinspector/quickinspector.cpp

```diff
@@ -243,9 +243,9 @@ static QString qsgTextureWrapModeToString(QSGTexture::WrapMode wrapMode)
 
 static bool isGoodCandidateItem(QQuickItem *item)
 {
-    if (!item->isVisible() || item->z() < 0 || qFuzzyCompare(item->opacity() + 1.0, qreal(1.0)) ||
```

Comment:
The problem is that depending the project those kind of items are assumed *background* / *not interesting* items while picking. I also find them wrong because for me looks like wrong written code. So let remove and we will see if somebody complains or not.